llvm/test/Transforms/LoopInterchange/deep-loop-nest.ll

```diff
@@ -0,0 +1,95 @@
+; REQUIRES: asserts
```

Comment:
Sure but in this case, I will have to duplicate the file for without asserts builds. I don't know if we can conditionally execute RUN line based on build type.

Reply:
I don't see why you need to duplicate the file? I think you only need to check optimization remarks instead of debug output as what you are doing now, e.g., `llvm/test/Transforms/LoopInterchange/loop-interchange-optimization-remarks.ll`.